lib/fittings/swagger_render.js

```diff
@@ -6,6 +6,7 @@ var injector = require('../../index.js').injector;
 var _ = injector.get('_');
 
 module.exports = function create(fittingDef) {
+    injector = require('../../index.js').injector;
```

Comment:
But I know the require has cache (see https://nodejs.org/api/modules.html#modules_caching), so the two require operation should return the same object, so I think this line will not refresh the injector